ci/environment-dev-3.6.yml

```diff
@@ -22,7 +22,7 @@ dependencies:
   - numpy
   - pandas
   - scipy
-  - xarray>=0.16.0
+  - xarray=0.16.0
```

Comment:
instead I could also require xs master

Reply:
We could also make a minor release on `xskillscore` with the fix so it stays compatible with the newest `xarray`.